applications/launchpad_v2/src/styles/index.ts

```diff
@@ -0,0 +1,17 @@
+import palette from './colors'
+import { gradients } from './gradients'
+
+const theme = {
+  colors: {
+    dark: palette.colors.dark,
+    light: palette.colors.light,
+    tari: palette.colors.tari,
+    monero: palette.colors.monero,
+    merged: palette.colors.merged,
+    secondary: palette.colors.secondary,
+    darkMode: palette.colors.darkMode
```

Comment:
how is this going to be used?
I imagined for 'dark mode' we would have a separate theme with exactly the same properties, but different color values, so that components are not aware what theme is being used - they just use provided properties

so in this case there would be a separate "darkTheme" with
```
colors: {
    dark: palette.colors.dark,
    light: palette.colors.light,
    tari: palette.colors.tari,
    monero: palette.colors.monero,
    merged: palette.colors.merged,
    secondary: palette.colors.secondary,
}
```